pkg/util/stringutil/string.go

```diff
@@ -5,8 +5,10 @@
 package stringutil
 
 import (
+	"openpitrix.io/notification/pkg/gerr"
```

Comment:
Should not use "openpitrix.io/notification/pkg/gerr" in util package.
May cause loop rely.